src/Symfony/Component/Debug/FatalErrorHandler/UndefinedFunctionFatalErrorHandler.php

```diff
@@ -54,7 +54,7 @@ public function handleError(array $error, FatalErrorException $exception)
         }
 
         $candidates = array();
-        foreach (get_defined_functions() as $type => $definedFunctionNames) {
+        foreach (get_defined_functions(false) as $type => $definedFunctionNames) {
```

Comment:
the function takes no arguments, why this here?

Reply:
This might be a false positive, it takes an optional param since 7.0.15: http://php.net/manual/en/function.get-defined-functions.php#refsect1-function.get-defined-functions-parameters

Reply:
ok, should be removed imho as it's the default value, and may break BC

Reply:
Sure, I'll report it to PHPStan, don't know why it was asked for here.